yoti-sdk-impl/src/main/java/com/yoti/api/client/spi/remote/call/aml/SimpleAmlAddressFactory.java

```diff
@@ -0,0 +1,12 @@
+package com.yoti.api.client.spi.remote.call.aml;
+
+import com.yoti.api.client.aml.AmlAddress;
+import com.yoti.api.client.aml.AmlAddressFactory;
+
+public class SimpleAmlAddressFactory implements AmlAddressFactory {
+
+    public AmlAddress create(String postCode, String country) {
```

Comment:
do we need another one with country only?

Reply:
Same as the other answer - I assume people will use the builder